testapps/setup_testapp_python3_sqlite_openssl.py

```diff
@@ -0,0 +1,30 @@
+
+from distutils.core import setup
+from setuptools import find_packages
+
+options = {'apk': {'requirements': 'libffi,openssl,sqlite3,requests,peewee,sdl2,pyjnius,kivy,python3',
+                   'android-api': 27,
+                   'ndk-api': 21,
+                   'dist-name': 'bdisttest_python3_sqlite_openssl_googlendk',
+                   'ndk-version': '10.3.2',
+                   'arch': 'armeabi-v7a',
+                   'permissions': ['INTERNET', 'VIBRATE'],
+                   }}
+
+package_data = {'': ['*.py',
+                     '*.png']
+                }
+
+packages = find_packages()
+print('packages are', packages)
+
+setup(
+    name='testapp_python3_sqlite_openssl_googlendk',
+    version='1.1',
+    description='p4a setup.py test',
+    author='Alexander Taylor',
```

Comment:
@KeyWeeUsr the sqlite test app isn't new though, just the setup file is (that is also why the `peewee` use isn't new, I missed it also on first read)

Reply:
> I think you should use own name/contact info ...

@KeyWeeUsr, I did it that way because it's almost a copy of the testap by @inclement, I have no problem to put my info in there...if @inclement wants that I will change it I will do it, no problem.